### PR TITLE
Clean up JS integrations code samples and intro lang

### PR DESCRIPTION
**Pull request by PeloWriter**

### CODE REVIEW

File: src/platforms/node/common/integrations/pluggable-integrations.mdx [hunk 1]
```diff
@@ -53,7 +53,7 @@ This integration extracts all non-native attributes from the Error object and at
 
 Available options:
 
-```js
+```JavaScript
 {
   depth: number; // limit of how deep the object serializer should go. Anything deeper than the limit will be replaced with standard Node.js REPL notation of [Object], [Array], [Function] or primitive value. Defaults to 3.
```
Comment (lobsterkatie) [inline src/platforms/node/common/integrations/pluggable-integrations.mdx:58]:
```suggestion
   // limit of how deep the object serializer should go. Anything deeper than limit will 
   // be replaced with standard Node.js REPL notation of [Object], [Array], [Function] or 
   // a primitive value. Defaults to 3.
  depth: number;
```

File: src/platforms/node/common/integrations/pluggable-integrations.mdx [hunk 2]
```diff
@@ -70,7 +70,7 @@ For example `C:\\Program Files\\Apache\\www` won't work, however, `/Program File
 
 Available options:
 
-```js
+```JavaScript
 {
   root: string; // root path that will be appended to the basename of the current frame's url
```
Comment (lobsterkatie) [inline src/platforms/node/common/integrations/pluggable-integrations.mdx:75]:
```suggestion
  // root path that will be appended to the basename of the current frame's url 
  root: string; 
  
```

File: src/platforms/node/common/integrations/pluggable-integrations.mdx [hunk 3]
```diff
@@ -70,7 +70,7 @@ For example `C:\\Program Files\\Apache\\www` won't work, however, `/Program File
...
   iteratee: (frame) => frame); // function that takes the frame, applies any transformation on it and returns it back
```
Comment (lobsterkatie) [inline src/platforms/node/common/integrations/pluggable-integrations.mdx:76]:
```suggestion
  // function that takes the frame, applies a transformation, and returns it
  iteratee: (frame) => frame); 
```

File: src/platforms/node/guides/express/index.mdx [hunk 4]
```diff
@@ -63,7 +63,7 @@ app.get("/debug-sentry", function mainHandler(req, res) {
 
 Possible options are:
 
-```js
+```JavaScript
 // keys to be extracted from req
 request?: boolean | string[]; // default: true = ['cookies', 'data', 'headers', 'method', 'query_string', 'url']
 // server name
```
Comment (lobsterkatie) [inline src/platforms/node/guides/express/index.mdx:69]:
```suggestion

// server name
```

I'd do the rest here but they're not close enough to what you changed for it to be willing to let me make suggestions there. Personally I think spacing it out makes it much easier to read, but you'll have to do it locally if you want to do it.

Comment (PeloWriter) [reply src/platforms/node/guides/express/index.mdx:69]: Done.

**Pull request merged**